src/Native/build.proj

```diff
@@ -44,7 +44,7 @@
 
     <!-- Run script that invokes Cmake to create VS files, and then calls msbuild to compile them -->
     <Message Text="$(MSBuildProjectDirectory)\build.cmd $(BuildArgs)" Importance="High"/>
-    <Exec Command="$(MSBuildProjectDirectory)\build.cmd $(BuildArgs)" />
+    <Exec Command="&quot;$(MSBuildProjectDirectory)\build.cmd&quot; $(BuildArgs)" />
```

Comment:
It appears the unix build has the same issue.  Would it be possible to fix the Unix build as well?

* build.sh
* run.sh
* https://github.com/dotnet/machinelearning/pull/196/files#diff-2cf0b368b22eef1d69c1c746867c3f56R33

Reply:
Certainly, I have some time in the end of the week. Is it okay if I will put this in a new pull request?

Reply:
Yes, it is totally fine with me to put create another PR for that.